adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory.Platform/Microsoft.IdentityModel.Clients.ActiveDirectory.Platform.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="MSBuild.Sdk.Extras/1.5.4">
```

Comment:
Oren mentioned that you can create a global.json file next to the sln where you can instruct on the version to use: https://github.com/onovotny/MSBuildSdkExtras (search for global.json)

Reply:
as mentioned for another change... reverting this change back as I couldn't get it to work properly from the commandline. will try again later.